packages/graph/hash_graph/lib/graph/src/api/rest/data_type.rs

```diff
@@ -129,6 +133,13 @@ async fn create_data_type<P: StorePool + Send>(
         .map(Json)
 }
 
+#[derive(Deserialize, Component)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+struct DataTypeQuery {
+    query: Expression,
+    data_type_query_depth: u8,
```

Comment:
Can you add doc comments to these explaining what it means?

Reply:
37bf7d989068202b6dd0bcc6c47edce48cf8269e